global_costmap:
  global_frame: world
  robot_base_frame: rexrov2/base_link
  
  update_frequency: 5.0
  publish_frequency: 2.0

  rolling_window: true
  
  width: 300.0
  height: 300.0
  resolution: 0.5
  
  plugins:
    - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
    - {name: inflation_layer, type: "costmap_2d::InflationLayer"}